site/content/docs/gsoc/_index.md

```diff
@@ -9,7 +9,7 @@ links:
 
   - name: '2024'
     link: 2024/
-    desc: Start getting involved now!
+    desc: Unfortunately ZAP will not be taking part
```

Comment:
...was not selected as a mentoring organisation


(Put it in Google, we wanted to 🙂)

Reply:
Same for any tweets you do, we tried but it didn't pan out. Not because of lack of want or trying on the ZAP team's part.

Reply:
Goog point, fixed